projects/storefrontapp/tsconfig.server.prod.json

```diff
@@ -290,6 +290,7 @@
       "@spartacus/cdc/core": ["../../dist/cdc/core"],
       "@spartacus/cdc": ["../../dist/cdc"],
       "@spartacus/cdc/root": ["../../dist/cdc/root"],
+      "@spartacus/cdc/manage-users": ["../../dist/cdc/manage-users"],
```

Comment:
please move this to the last of this array.

Reply:
done